test/export/StructureDefinitionExporter.test.ts

```diff
@@ -69,7 +69,63 @@ describe('StructureDefinitionExporter', () => {
     expect(exported.version).toBe('0.0.1');
     expect(exported.type).toBe('Observation');
     expect(exported.baseDefinition).toBe('http://hl7.org/fhir/StructureDefinition/Observation');
-    expect(exported.derivation).toBe('constraint');
+  });
+
+  it('should properly set/clear all metadata properties for a profile', () => {
+    const profile = new Profile('Foo');
+    profile.parent = 'Observation';
+    doc.profiles.set(profile.name, profile);
+    exporter.exportStructDef(profile);
+    const exported = pkg.profiles[0];
+
+    expect(exported.id).toBe('Foo'); // defaulted from user-provided name
+    expect(exported.meta).toBeUndefined();
+    expect(exported.implicitRules).toBeUndefined();
+    expect(exported.language).toBeUndefined();
+    expect(exported.text).toBeUndefined();
+    expect(exported.contained).toBeUndefined; // inherited from Observation
```

Comment:
I'm not sure if this makes a difference (but I noticed it thanks to GitHub syntax highlighting). This is a `toBeUndefined` the property rather than `toBeUndefined()`. I can see that causing issues with some libraries, so could you either fix this or explain why it is this way if it should be?

Reply:
You are right.  I missed the `()`.  Multiple times (copy/paste).  I'll fix it.